atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/cli/AtlasDbPerfCli.java

```diff
@@ -62,8 +65,9 @@
     @Arguments(description = "The performance benchmarks to run. Leave blank to run all performance benchmarks.")
     private List<String> tests;
 
-    @Option(name = {"-b", "--backend"}, description = "The underlying physical store to use e.g. 'POSTGRES'.")
-    private String backend;
+    @Option(name = {"-b", "--backends"}, description = "Space delimited list of backing KVS stores to use in " +
+            "quotes. (e.g. \"POSTGRES ORACLE CASSANDRA\")" + " Defaults to all backends if not specified.")
```

Comment:
remove ORACLE (since we don't support it and it won't run if you leave it blank)